### PR TITLE
Stage2 dev

### PR DESCRIPTION
**Pull request by JancyRoachR**

### CODE REVIEW

File: src/test/java/org/recap/request/GFAServiceIT.java [hunk 1]
```diff
@@ -95,9 +97,9 @@ public void testLasStatusCheck() throws Exception {
```
Comment (reachrama) [inline src/test/java/org/recap/request/GFAServiceIT.java:97]: Remove all commented out code

File: src/test/java/org/recap/camel/requestinitialdataload/processor/RequestInitialDataLoadProcessorUT.java [hunk 2]
```diff
@@ -107,15 +108,17 @@ public void processInput() throws ParseException {
...
         requestDataLoadCSVRecordList.add(requestDataLoadCSVRecord1);
         CamelContext ctx = new DefaultCamelContext();
+        ctx.setRouteController(routeController);
         Exchange exchange = new DefaultExchange(ctx);
         exchange.getIn().setBody(requestDataLoadCSVRecordList);
         exchange.getIn().setHeader("John", "CUL");
         exchange.getIn().setHeader("CamelAwsS3Key", "GEJGNSIKHIL");
-        exchange.getIn().setHeader("CamelAwsS3BucketName", "AWSS3Bucket/");
+        exchange.getIn().setHeader("CamelAwsS3BucketName", "AWSS3Bucket/Htsvmlfkgdr/fksifdg");
         exchange.setProperty("CamelSplitIndex",0);
```
Comment (reachrama) [inline src/test/java/org/recap/camel/requestinitialdataload/processor/RequestInitialDataLoadProcessorUT.java:117]: Is this  key hardcoded?

**Review by reachrama — COMMENTED** [review]
Please review the comments, provide a response.

**Review by srinduri04 — APPROVED** [review]
Approved

**Pull request merged**